### ISSUE TITLE
Check use of <md> tag

### ISSUE THREAD

**Issue by Pantagrueliste** [cu-mkp/m-k-manuscript-data#124]
check use of `<md>` tag; need not be used in entries already grouped as medical (as main category or subcategory)

**Comment by TillmannTaape** [comment]
To confirm, does this mean I should go through all 44 medical entries and remove and md tags?

What happened to our plans of having a medical-related sub-category? Does this have an impact on md tag use?

**Comment by TillmannTaape** [comment]
Team decision: we should try to tag things as medical within these entries so they come up when a user pulls all medical-tagged terms. Taks is now to figure out a markup protocol for these entries. E.g. can tag expansively; or limit tagging to certain categories such as diseases, cures, remedies.

**Comment by TillmannTaape** [comment]
Still working through, but here are my suggested categories to mark up thus far:

- **symptoms** (e.g. 7v "defluxion")
- **indication** (for/against x)
- **counter-indication** (don't do x – e.g. 165v, "If you have a cut on your hand,
take care that this mixture does not touch it"; don't touch your ear with anything bigger than your elbow)

- **cure/curative effect** (e.g. 1v "lascher le ventre", 15v "conforte for bien", 1v "remollitif")

- **negative effect** (e.g. "poison", 15v "they will be neither able to move, nor speak, nor thwart thieves")

QUESTION FOR THE TEAM:
- **procedures** (e.g. enema, injection)? – Could be difficult to demarcate in some cases (e.g. put it on the wound; put garlic in your ear)

- DON'T tag the remedy itself or its ingredients? Too many.

**Comment by TillmannTaape** [comment]
Finished marking up all three versions, as detailed above, including procedures such as enema and injection, but not more general applications such as anointing or making oneself sweat.

I did not encounter any major difficulties. Occasionally I made a judgment call to include the remedy with the disease in the md tag, when I thought it was particularly interesting, e.g. "water against the plague" as opposed to just "against the plague".
